### ISSUE TITLE
Lien vers la documentation officielle

### ISSUE THREAD

**Issue by nchachereau** [LausanneCitationStyle/Lausanne#17]
CSL nous avait demandé de mettre à jour le style avec le lien vers la documentation officielle une fois que celle-ci serait disponible. Puisqu'il faut faire une nouvelle pull request pour d'autres raisons (#16), est-ce que j'ajoute le lien vers la [documentation](https://www.unil.ch/hist/files/live/sites/hist/files/shared/Formations/Guide_Bibliographique_UNIL_HIST.pdf)?

Il vaudrait peut-être la peine de demander à Florian si on pourrait disposer d'une URL plus propre et plus stable pour la documentation, quelque chose comme `www.unil.ch/hist/guide-bibliographique`.

**Comment by grandjeanmartin** [comment]
Oui, il faut absolument qu'on se crée une telle page, sur laquelle on pourrait mettre le guide mais aussi rassembler les autres ressources (tutoriel Zotero, autres documents de bibliographie, etc.). Je regarde avec lui si je le croise la semaine prochaine.
